pkg/mtbroker/common.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package broker
+
+import (
+	"context"
+	"net/http"
+)
+
+const (
+	livenessURI  = "/healthz"
+	readinessURI = "/readyz"
+)
+
+type Sender interface {
+	NewCloudEventRequestWithTarget(ctx context.Context, target string) (*http.Request, error)
+	Send(req *http.Request) (*http.Response, error)
+}
+
+func WithLivenessCheck(next http.Handler) http.Handler {
```

Comment:
Maybe it's better to put these in kncloudevents package? or another package under knative.dev/pkg like knative.dev/pkg/health?